curiouslearning/FeedTheMonsterJS
# fix: Stacking volume issue in pause button (FM-329)

# Changes
- Fix volume stacking when pause button is clicked

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level
- In gameplay scene
- Hit pause and click restart or select map button
- Play the game again
- Repeat process
- Make sure volume is not stacking anymore

Ref: [FM-329](https://curiouslearning.atlassian.net/browse/FM-329)


[FM-329]: https://curiouslearning.atlassian.net/browse/FM-329?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
